spec/features/users/user_creates_account_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+feature 'user creates an account' do
+
+  before :each do
+    visit root_path
+    click_link "Sign Up"
+  end
+
+  scenario "user specifies valid and required info as a volunteer" do
+    user = FactoryGirl.create(:user)
+    # attach_file "Profile Photo",
+      #"#(Rails.root)/spec/support/images/soldier.jpg"
```

Comment:
Incorrect indentation detected (column 6 instead of 4).<br>Missing space after #.